tests/cargo-add.rs

```diff
@@ -23,6 +23,42 @@ fn adds_dependency() {
     assert_eq!(val.as_str().unwrap(), "CURRENT_VERSION_TEST");
 }
 
+fn upgrade_test_helper(upgrade_method : &str , expected_prefix : &str) {
+	let (_tmpdir, manifest) = clone_out_test("tests/fixtures/add/Cargo.toml.sample");
```

Comment:
Is this line indented differently? Looks weird in GitHub's diff; just rustfmt it :)